### PR TITLE
feat: Add Polish translation

### PR DESCRIPTION
**Pull request by verebes1**
## New language
This PR adds a Polish translation to the App.

Changes include:
- Add pl-PL.json including the new translation.
- Updated app_widget.dart with the new language entry
- Updated language.dart with the new language entry

### How to test changes. 
Build and run the project.
Open settings choose Polski and the app should show the correct translations.

### CODE REVIEW

**Comment by appflowy** [comment]
Hi @verebes1, Your commit message was not consistent with our [Commit message guidelines.](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/submitting-your-first-pull-request) Also I'm going to update this with the remote branch.

**Comment by appflowy** [comment]
@verebes1    Could you mind updating your commit message, because it's not the AppFlowy Style.

image

You could check out this [guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/submitting-your-first-pull-request#commit-message-guidelines) for more information.

**Comment by verebes1** [comment]
Will try today or tomorrow

**Comment by verebes1** [comment]
@appflowy Updated the commit and squashed it into one 👍

File: frontend/app_flowy/assets/translations/pl-PL.json [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+    "passwordHint": "Password",
+    "dontHaveAnAccount": "Nie masz konta?",
+    "repeatPasswordEmptyError": "Powtórzone hasło nie moze być puste",
+    "unmatchedPasswordError": "Hasła nie są takie same"
+  },
+  "workspace": {
+    "create": "Utwórz przestrzeń",
+    "hint": "przestrzeń robocza",
+    "notFoundError": "Przestrzeni nie znaleziono"
+  },
```
Comment (MikeWallaceDev) [inline frontend/app_flowy/assets/translations/pl-PL.json:36]: Here, you seem to be using "space" instead of "workspace". Are you sure this is ok?

**Review by MikeWallaceDev — COMMENTED** [review]
I don't know Polish, but Google Translate helped me out :)  I'll merge this immediately if you can reply quickly...

File: frontend/app_flowy/assets/translations/pl-PL.json (hunk 1 above)
Comment (verebes1) [reply frontend/app_flowy/assets/translations/pl-PL.json:36]: Hey Mike,
Yes, I used space on purpose here as the workspace would be "przestrzeń robocza" which might be too long and if it is, there as a hint. But don't worry, it is pretty much interchangeable in Polish. People know, depending on the context, what space will mean. If not, there is always the hint :)

Comment (appflowy) [reply frontend/app_flowy/assets/translations/pl-PL.json:36]: Thank you for explaining!

**Pull request merged**